ccd-definition/FixedLists/CareSupervision/hearing-cancellation/HearingCancellationReasons-Other.json

```diff
@@ -51,21 +51,21 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "HearingCancellationReasons-Other",
-    "ListElementCode": "OT8",
+    "ListElementCode": "1NT",
     "ListElement": "Immigration and international difficulties",
     "DisplayOrder": 8
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "HearingCancellationReasons-Other",
-    "ListElementCode": "OT9",
+    "ListElementCode": "OT8",
```

Comment:
need to be careful with this, may lead to problems
1. this gets deployed on prod
2. user adjourn hearing with reason 'Severe weather' - OT8
3. later migration is run and change OT8-> IN1
The result is that user select 'Severe weather' but it is recorded as 'Immigration and international difficulties'